website/docs/r/iam_saml_federation_v1.html.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: "selectel"
+page_title: "Selectel: selectel_iam_saml_federation_v1"
+sidebar_current: "docs-selectel-resource-iam-saml-federation-v1"
+description: |-
+  Creates and manages SAML Federation for Selectel products using public API v1.
+---
+
+# selectel\_iam\_saml\_federation\_v1
+
+Manages SAML Federation for Selectel products using public API v1.
+Selectel products support Identity and Access Management (IAM).
+For more information about Federations, see the [official Selectel documentation](https://docs.selectel.ru/en/control-panel-actions/users-and-roles/federations/).
+
+## Example Usage
+
+```hcl
+resource "selectel_iam_saml_federation_v1" "federation_1" {
+  name                  = "federation name"
+  description           = "simple description"
+  issuer                = "http://localhost:8080/realms/master"
+  sso_url               = "http://localhost:8080/realms/master/protocol/saml"
+  sign_authn_requests   = true
+  force_authn           = true
```

Comment:
Is it common practice to list all available options in an example?
I would either set default values for these two fields or remove them from the example.

Reply:
Agree, I removed these optional params, only description left